using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -211,17 +322,6 @@ The result for the `getUrlVariables` request:
 The Analytics Custom VID is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/987092ee0fd2699ae5898755cd72255802628107/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
 {% endhint %}
 
-### Configuration content request
```

Comment:
I would suggest keeping this order for dispatched events, based on their importance and relation to the Identity extension:
Identity response identity
Configuration Content request
Analytics content request

Reply:
Let's revert this deletion and keep the following order: 
Identity response identity
Configuration content request
Analytics content request

Reply:
It is not deleted though 
Ref : https://github.com/samthiru/aep-sdks-documentation/blob/AMSDK-9006/using-mobile-extensions/mobile-core/identity/identity-event-reference.md